Tasks/DownloadPackageV1/package.ts

```diff
@@ -0,0 +1,185 @@
+var fs = require("fs");
+var path = require("path");
+
+import * as tl from "vsts-task-lib/task";
+
+import { Extractor } from "./extractor";
+import { PackageUrlsBuilder } from "./packagebuilder";
+import { WebApi } from "azure-devops-node-api";
+import { VsoClient } from "azure-devops-node-api/VsoClient";
+import { ICoreApi } from "azure-devops-node-api/CoreApi";
+import stream = require("stream");
+
+export class Result {
```

Comment:
The name Result here kept throwing me when I was looking at other code - how about PackageFile, and changing Value to Location

Reply:
I changed the Result class to PackageFileResult, the Value field can contain either a URL or the content of the file, so location seems odd